app/frontend/src/components/DateField.stories.tsx

```diff
@@ -8,10 +10,23 @@ export default {
   title: "Components/Simple/DateField",
 } as Meta;
 
-const Template: Story<any> = (args) => (
-    <>
-      <DateField {...args} />
-    </>
+const Template: Story<any> = (args) => {
+  const { control, register } = useForm();
+  return (
+    <DateField
+      control={control}
+      error={false}
+      helperText=""
+      id="date-field"
+      inputRef={register({
+        required: "Enter your birthdate",
+        validate: (stringDate) =>
+          validatePastDate(stringDate) || "Must be a valid date in the past.",
+      })}
```

Comment:
```suggestion
      inputRef={register}
```
and remove the unused import